packages/edit-site/src/components/global-styles/style.scss

```diff
@@ -85,23 +85,35 @@
 
 .edit-site-global-styles-variations_item {
 	box-sizing: border-box;
+	// To round the outline in Windows 10 high contrast mode.
+	border-radius: $radius-block-ui;
 
 	.edit-site-global-styles-variations_item-preview {
 		padding: $border-width * 2;
 		border-radius: $radius-block-ui;
-		border: $gray-200 $border-width solid;
+		box-shadow: 0 0 0 $border-width $gray-200;
+		// Shown in Windows 10 high contrast mode.
+		outline: 1px solid transparent;
```

Comment:
Nitpick we have this for other high contrast mode contexts:

```
// Windows high contrast mode.
outline: 2px solid transparent;
```

Completely separate, I wonder if we should create a mixin for this, then the mixin can contain the named context so we don't need a CSS comment, and it can be consistent across.